File: 코딩테스트/2019 카카오 겨울 인턴십/2.js
function solution(s) {
  s = s.slice(2, s.length - 2);
  s = s.split("},{");
  dict = {};
  for (let i in s) {
    numArr = s[i].split(",");
    for (let j in numArr) {
      if (dict[numArr[j]]) {
        dict[numArr[j]] += 1;
      } else {
        dict[numArr[j]] = 1;
      }
    }
  }
  sortDict = [];
  for (let i in dict) {
    sortDict.push([dict[i], i]);
  }
  sortDict.sort((a, b) => b[0] - a[0]);
  return sortDict.map((ele) => Number(ele[1]));
}

console.log(solution("{{2},{2,1},{2,1,3},{2,1,3,4}}"));

File: 코딩테스트/2019 카카오 겨울 인턴십/2.py
from collections import defaultdict

def solution(s:str):
  s = s[2:-2].split("},{")
  count = defaultdict(int)
  for ele in s:
    for num in ele.split(","):
      count[int(num)] += 1
  print(count)

  return [key for key,_ in sorted(count.items(), key = lambda x : -x[1])]

print(solution("{{4,2,3},{3},{2,3,4,1},{2,3}}"))